elf/elf.go

```diff
@@ -674,6 +695,17 @@ func (b *Module) Load(parameters map[string]SectionParams) error {
 					fd:    int(progFd),
 				}
 			}
+
+			if params, ok := parameters[secName]; ok && params.CustomConfig != nil {
+				for _, cc := range params.CustomConfig {
+					if isPinReserved(cc.Id) {
+						continue
+					}
+					if err := PinObjectCustom(int(progFd), cc.PinPath); err != nil {
```

Comment:
We should pin a bpf object only one time and not pin it several time.

Here you pin a bpf program and not a bpf map. But the custom pinning parameter has to be correlated with the `pinning` field in the bpf_map_def`. We have no equivalent `struct bpf_prog_def` at all at the moment, so we can't decide where to pin a program... I would just not pin bpf programs at all at the moment, as per previous comment.